package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "google-map-react",
-  "version": "1.0.5",
+  "name": "@jfw/google-map-react",
+  "version": "1.0.6",
```

Comment:
Lets not make a version from this PR, version will be bumped when necessary with one or multiple PRs

Reply:
Same here.